lib/menu.ts

```diff
@@ -6,24 +6,24 @@
  * entirely by a custom item. For an item to be a selectable menu item, it needs `tabindex=-1`
  * attribute set. If unset, or if the "disabled" class is set, the item will not be selectable.
  *
- * Further, if `dom.data('menuItemSelected', (yesNo: boolean) => {})` is set, that callback will be
- * called whenever the item is selected and unselected. It may call onMenuItemSelected(yesNo) to keep
- * the default behavior of getting the suitable css class. A callback should be seldom needed, but it
- * is needed for nested menus.
+ * Further, if `dom.data('menuItemSelected', (yesNo: boolean, elem) => {})` is set, that callback
+ * will be called whenever the item is selected and unselected. In addition, the selected item
+ * gets a css class. A callback should be seldom needed, but it is use for nested menus.
```

Comment:
The focus logic changed a bit, and I decided that it makes the usage simpler to always the -sel css class, and skip the exposing of `onMenuItemSelected()` function.